configure.ac

```diff
@@ -34,6 +34,22 @@ AC_PROG_RANLIB
 
 AM_MISSING_PROG([GPERF], [gperf])
 
+GPERFVER=`gperf -v|head -1 |cut -d " " -f3`
+if test "$GPERFVER"x != x; then
+  GPERF_MAJOR_VER=`echo $GPERFVER | cut -f1 -d.`
+  GPERF_MINOR_VER=`echo $GPERFVER | cut -f2 -d.`
+  if test  $GPERF_MAJOR_VER -gt 3 -o $GPERF_MAJOR_VER -eq 3 -a $GPERF_MINOR_VER -ge 1 ; then
+    GPERF_SIZE_TYPE="size_t"
+  else
+    GPERF_SIZE_TYPE="unsigned int"
+  fi
+else
+  GPERF_SIZE_TYPE="unsigned int"
+fi
+
+AC_DEFINE_UNQUOTED(GPERF_SIZE_TYPE, $GPERF_SIZE_TYPE)
+AH_TEMPLATE([GPERF_SIZE_TYPE], [GPERF size type])
+
```

Comment:
I could not find any.

Reply:
Could you please cut and paste the test code into a macro file in m4/gprefver.m4 perhaps?  This will help to keep the configure.ac file more concise.  You can find some tips here too:
https://autotools.io/autoconf/macros.html
https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/html_node/Writing-Autoconf-Macros.html#Writing-Autoconf-Macros

Reply:
done
Travis still has trouble finding some apt packages :(